portal-web/test/functional/com/liferay/portalweb/macros/FormPortletNavigator.macro

```diff
@@ -1,5 +1,11 @@
 definition {
 
+	macro exitWidgetModal {
+		SelectFrameTop();
+
+		Click(locator1 = "FormPortlet#EXIT_MODAL");
```

Comment:
@evanilson-santana I believe SF failed because you forgot to change to WIDGET_MODAL_CLOSE_BUTTON.

Reply:
Yes, I realised. I accidentally sent it without seeing this detail